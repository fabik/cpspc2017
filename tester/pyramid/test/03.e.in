7 50
1547791 1382 39 29469 6 6 2
419635721100207467
168016243897522567
149516393804716472
490683295420125621
322257635876893771
420432472283556786
165155462654478255
27557694966430370
35753328252323157
506938549892862466
52773665999042919
484108159533986098
329165958308696926
120360514979353199
404226260365080742
164365974941561307
461934100383612019
161159782120157796
162353915361033858
24905895181498547
368504028082425348
51641216028524623
504504082092440352
335764576588772635
150862714811793925
420427228762328095
396212440650199814
98454071605852616
232651950690864076
280984392328590723
458919559913342978
43048832446692581
43424642809860782
114801287464613911
39376510338604950
440053582743740213
393305914373859408
371115880339397899
86569482164772716
295560108317537175
115254818889267653
356768594858736438
220603590817066230
8322139522788631
12427195188567959
49082110794144204
9809570559923377
450483415698178414
414858150353210816
98290351291987572
