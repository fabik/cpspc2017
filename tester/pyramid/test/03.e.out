818264163676665128
327621468774034078
291547884941263050
956802617528311734
628382813116753488
819817780082323880
322043119139822286
53735831080924734
69716818111531418
988499376258985026
102905442748394192
943981501983556634
641853622184788660
234695692423560172
788216651353170880
320503666060257772
900743433693214466
314251784831668593
316580272108924536
48564982594608570
718560468493441178
100697234101136208
983752311939752588
654720526971186350
294173129117350458
819807555534466832
772590189741036752
191979458640718810
453657170463676494
547902495507636555
894865262906640460
83942608095649022
84675415473706858
223855536487573528
76781803074934192
858077753515985700
766922640070864604
723653421727010876
168805231208055758
576324256225275252
224739895214048010
695677086623592994
430163600636110614
16227666507420676
24232275677484528
95707134357289988
19128066669820424
878415294174762750
808948190952875522
191660213974534468
